crypto/cipher/aes256cbc/encrypter.go

```diff
@@ -44,7 +44,7 @@ type Encrypter struct {
 func (e Encrypter) Encrypt(message mc.PlainContent) (mc.EncryptedContent, error) {
```

Comment:
All errors returned by this function should be `mc.ErrEncrypt`

Reply:
done